Sources/App/Core/Score.swift

```diff
@@ -82,14 +87,21 @@ enum Score {
             let defaultVersion = versions.latest(for: .defaultBranch),
             let repo = package.repository
         else { return 0 }
-
+        
+        let hasDocumentation = [
+            defaultVersion,
+            versions.latest(for: .release),
+            versions.latest(for: .preRelease)
+        ].compactMap { $0 }.documentationTarget() != nil
```

Comment:
I've just had a look and realised that we can push `hasDocumentation()` up from `PackageResult` to `[Version]` and then simply reuse it here. `PackageResult.documentationTarget()` isn't actually being used anywhere so this is even simpler overall.

I'll push this in a PR to clarify.